packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx

```diff
@@ -21,6 +21,55 @@ type DomainListProps = GetDomainsParams & {
   fallback?: React.ReactNode;
 };
 
+const useDomainList = () => {
+  const { isAuthorizedUser: canDeleteDomain } = useGate({ permission: 'org:sys_domains:delete' });
```

Comment:
❓ What else does the `useGate` return? I'd expect to the DX to be like `const canDeleteDomain = useGate({ permission: 'org:sys_domains:delete' });` for the internal and external gate hook.

Reply:
At this stage of implementation this request is reasonable. We decides on this signature because in the future useGate may actually perform a request, which will require more properties to be returned. For example if in the future we depend on CF workers for authorization the following signature makes more sense
```tsx
const { isAuthorizedUser, isLoading, isError } = useGate({ permission: 'org:sys_domains:delete' });
```